src/components/drag-and-drop/drag-and-drop.stories.mdx

```diff
@@ -0,0 +1,129 @@
+import { useState } from 'react';
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import {
+  DraggableContext,
+  WithDrop,
+  WithDrag
+} from "./";
+
+import {
+  Table,
+  TableHeader,
+  TableRow,
+  TableCell
+} from '../table';
+
+<Meta
+  title="DraggableContext"
+  parameters={{ info: { disable: true }}}
+/>
+
+# DraggableContext
+A draggable context component
+
+## Related Components
+Although the Table component has drag and drop enabled already, any combination of components can be made draggable through the use of the WithDrag and WithDrop components:
+
+## Quick Start
+
+A draggable context is used to define an area in the page where drag and drop can be used.
+
+You also need to use WithDrop and WithDrag
+
+```jsx
+import {
+  DraggableContext,
+  WithDrop,
+  WithDrag
+} from "carbon-react/lib/components/drag-and-drop";
+
+const MyComponent = () => (
+  return (
+    <DraggableContext onDrag={ onItemMoved }>
+      <ol>
+        {items.map((item, index) => {
+          return (
+            <WithDrop index={ index }>
+              <li>
+                <WithDrag><span>{ item.content }</span></WithDrag>
+              </li>
+            </WithDrop>
+          );
+        })}
+      </ol>
+    </DraggableContext>
+  )
+);
+```
+
+### Basic 
+<Preview>
```

Comment:
This story doesn't appear on the sidebar in Storybook: 
![image](https://user-images.githubusercontent.com/14963680/96451428-76179b00-120f-11eb-8e45-3463979a69ec.png)

I think maybe because `<Story />` has no children but that's a bit of a guess. It's still there on the docs page at least

Reply:
I was just about to comment the same thing but you beat me to it 😉

Reply:
@sjohnson-sage @DipperTheDan 
That's because I've actually create the story in `drag-and-drop.stories.js` and just referenced it through the `id` prop in `drag-and-drop.stories.mdx`

I've done it that way because it seems that all elements created in MDX are wrapped by `MDXCreateElement` which is not supported by the legacy `react-dnd` resulting in this error:
`Only native element nodes can now be passed to React DnD connectors.You can either wrap MDXCreateElement into a <div>, or turn it into a drag source or a drop target itself.`

Reply:
Ah ok, maybe worth raising a Jira to do something about that in the future then?